test/NuGet.Core.FuncTests/Dotnet.Integration.Test/DotnetAddPackageTests.cs

```diff
@@ -29,7 +29,7 @@ public DotnetAddPackageTests(MsbuildIntegrationTestFixture fixture)
         [Fact]
         public async Task AddPkg_V3LocalSourceFeed_WithRelativePath_NoVersionSpecified_Success()
         {
-            using (var pathContext = new SimpleTestPathContext())
+            using (var pathContext = _fixture.CreateSimpleTestPathContext())
```

Comment:
Because the type is no longer obvious, `var` should be changed to `SimpleTestPathContext` here and elsewhere.